File: dev-docs/snippets/group-type-configs/delete.swift
import JsBaoClient

// Delete a group type configuration. Returns the typed `SuccessResult`
// (`{ success }`). The `groupType` path segment is percent-encoded (#590).
func delete(client: JsBaoClient, groupType: String) async throws {
  // #region example
  let result = try await client.groupTypeConfigs.delete(groupType: groupType)
  let success = result.success
  // #endregion example
  _ = success
}

File: dev-docs/snippets/group-type-configs/delete.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Delete a group type configuration. Resolves to `{ success: boolean }`.
export async function deleteConfig(client: JsBaoClient, groupType: string) {
  // #region example
  const { success } = await client.groupTypeConfigs.delete(groupType);
  // #endregion example
  return success;
}
